src/amo/reducers/home.js

```diff
@@ -226,20 +238,14 @@ type LoadHomeDataAction = {|
 |};
 
 export const loadHomeData = ({
-  collections,
-  heroShelves,
+  homeShelves,
   shelves,
```

Comment:
I'm not really a huge fan of keeping `shelves` around, but I can see why you're doing it. I suppose it is the lesser evil, so let's leave it for now, unless I can think of a different, better way.